lib/msal-core/azure-pipelines.yml

```diff
@@ -0,0 +1,29 @@
+# Node.js
+
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+jobs:
+  - job: rolling_VS2017_build
+    displayName: 'Build Job for JS (VS2017)'
```

Comment:
what is vs 2017

Reply:
Visual Studio 2017

Reply:
This is the server that we are using to build.